tile_generator/templates/jobs/spec

```diff
@@ -115,7 +115,7 @@ consumes:
 {% for link_name, link in release.consumes.iteritems() %}
   - name: {{ link_name }}
     type: {{ link.type or link_name }}
-{% if link.optional %}
+{% if 'optional' in link and link.optional %}
```

Comment:
`if link.optional is defined` http://jinja.pocoo.org/docs/2.10/templates/#builtin-tests and https://stackoverflow.com/questions/33400771/check-if-key-exists-in-a-dict-in-jinja2-template-on-ansible/33444226